proxy/src/telemetry/metrics/mod.rs

```diff
@@ -120,6 +124,7 @@ pub struct Aggregate {
 #[derive(Debug, Clone)]
 pub struct Serve {
     metrics: Arc<Mutex<Metrics>>,
+    handle: reactor::Handle,
```

Comment:
afaict, this is unused

Reply:
removed in ae253d7d71f64892026104b9e5cc482a26fe0536